### ISSUE TITLE
Timestamps should be relative

### ISSUE THREAD

**Issue by bentona** [Originate/replay-ruby#1]
https://github.com/Originate/replay-ruby/blob/master/lib/replay_api/event_properties.rb#L23

**Comment by bentona** [comment]
@alexpeachey @jimmzhu wanted to get your input here. Timestamps are processed as an offset currently, since we're looking to imitate GA's approach to ensure accuracy.(https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#qt). However, I'm wondering if we shouldn't just allow absolute timestamps, and discourage users from using them unless they need to, so we can just stamp them on receipt. 
Either way, I can see why `timestamp` is a misleading name for an attribute that is supposed to be relative.

**Comment by alexpeachey** [comment]
Yeah, I think we were originally thinking we wanted everything to be stamped on the server and the relative timestamp thingy would adjust it with the idea being people would say this thing happened "X time period ago" but you are right most people likely won't be saying it happened "X time period ago" and instead will be working with event data they may already have with timestamps and just want to send the timestamp.

So if you changed to timestamp expecting the actual timestamp in seconds since epoch that is probably fine.

**Comment by bentona** [comment]
Ok, sounds good. As part of that (and as an interim fix), I'd like the timestamp to not be included in requests from the gem by default, should we just remove it from the above file?
